pages/index.tsx

```diff
@@ -28,14 +29,46 @@ import {
   Synth,
 } from "../utils/umaApi";
 
-import { getDefillamaTvl, getDefillamaPercentChange } from "../utils/defillama";
+import {
+  getDefillamaTvl,
+  getDefillamaPercentChange,
+  getDefillamaStats,
+} from "../utils/defillama";
+import { ethers } from "ethers";
+
+async function attachDefillamaStats(cmsSynth: ContentfulSynth) {
```

Comment:
nit: the type here is slightly unsafe as "externalApi" might be undefined, I think you want a ContentfulSynth & {externalApi: string}

Reply:
i wonder why typescript didnt error

Reply:
yeah, I preferred to add an extra assertion to check for the existence of the `externalApi`. Modifying the type is causing some build errors, requiring for other types to be changed too.